src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1783,6 +1861,19 @@ static void glb_update_locked(grpc_exec_ctx *exec_ctx, grpc_lb_policy *policy,
         &glb_policy->lb_channel_connectivity,
         &glb_policy->lb_channel_on_connectivity_changed, NULL);
   }
+
+  // Propagate update to fallback_backend_addresses if we haven't heard from 
+  // the balancer.
+  if (glb_policy->serverlist == NULL) {
+    grpc_lb_addresses *backends = grpc_lb_addresses_extract_backends(addresses);
```

Comment:
This can just be:

```
if (glb_policy->fallback_backend_addresses != NULL) {
  grpc_lb_addresses_destroy(exec_ctx, glb_policy->fallback_backend_addresses);
}
glb_policy->fallback_backend_addresses = grpc_lb_addresses_copy(addresses);
```